### DISCUSSION TITLE
How to upload image on cloudinary

### DISCUSSION THREAD

**Discussion by kai0310** [filamentphp/filament#3133] [Help]
Hey there 👋 

I know how to use [file upload](https://filamentphp.com/docs/2.x/forms/fields#file-upload) to AWS S3 or publicly to your default storage disk, but I don't know how to upload to external media such as cloudinary.

I would like to use [Cloudinary](https://cloudinary.com/) in my case. Is there any way to do this?

## What I did
Using [codebar-ag/laravel-flysystem-cloudinary
](https://github.com/codebar-ag/laravel-flysystem-cloudinary)

1. Registration to the Laravel file system

```PHP
'cloudinary' => [
    'driver' => 'cloudinary',
    'cloud_name' => env('CLOUDINARY_CLOUD_NAME'),
    'api_key' => env('CLOUDINARY_API_KEY'),
    'api_secret' => env('CLOUDINARY_API_SECRET'),
    'url' => [
        'secure' => (bool) env('CLOUDINARY_SECURE_URL', true),
    ],
],
```
2. Upload using file system with `FileUpload`
```PHP
FileUpload::make('image')->disk('cloudinary') # it's not work 🔥
```

**Comment by danharrin** [comment]
We use Livewires file upload system, so if you add support to Livewire for that driver then it should work

**Reply by kai0310** [reply]
Thank you!

I was indeed not able to change this, I tried to change this but it did not work.
I will investigate to see if there is some other logic that may be causing the problem. If I find out anything, I will mention it here :)
